test/lib/ufe/testRename.py

```diff
@@ -270,29 +270,6 @@ def testRenameRestrictionSameLayerDef(self):
             newName = 'Ball_35_Renamed'
             cmds.rename(newName)
 
-    def testRenameRestrictionOtherLayerOpinions(self):
-        '''Restrict renaming USD node. Cannot rename a prim with definitions or opinions on other layers.'''
-
-        # select a USD object.
-        mayaPathSegment = mayaUtils.createUfePathSegment('|world|transform1|proxyShape1')
-        usdPathSegment = usdUtils.createUfePathSegment('/Room_set/Props/Ball_35')
-        ball35Path = ufe.Path([mayaPathSegment, usdPathSegment])
-        ball35Item = ufe.Hierarchy.createItem(ball35Path)
-
-        ufe.GlobalSelection.get().append(ball35Item)
-
-        # get the USD stage
-        stage = mayaUsd.ufe.getStage(str(mayaPathSegment))
-
-        # set the edit target to Assembly_room_set.usda
-        stage.SetEditTarget(stage.GetLayerStack()[2])
-        self.assertEqual(stage.GetEditTarget().GetLayer().GetDisplayName(), "Assembly_room_set.usda")
-
-        # expect the exception happens
-        with self.assertRaises(RuntimeError):
-           newName = 'Ball_35_Renamed'
-           cmds.rename(newName)
```

Comment:
testRenameRestrictionOtherLayerOpinions no longer makes any sense.  Removing it all together.

Reply:
Why does it no longer make sense?

Reply:
@ppt-adsk 
What this test was doing was to exercise the following:

"A prim composed of multiple primSpecs can not be renamed". 

So in the case of '/Room_set/Props/Ball_35' in Assembly_room_set.usda

```
def "Ball_35" (
    add references = @./Ball.usd@
    variants = {
        string shadingVariant = "Ball_8"
    }
)
{
    double3 xformOp:translate = (6.43, 20.0025, 0)
    uniform token[] xformOpOrder = ["xformOp:translate"]
}
```
Ball_35 is a result of composition from different primSpecs:

Assembly_room_set.usda	  ------>  root
Ball.usd                  ------>  reference
Ball.shadingVariants.usda ------>  reference
Ball.shadingVariants.usda ------>  variant
Ball.maya.usd             ------>  reference

This was the third rule that we previously had and was very restrictive. 

one should be able to rename Ball_35 because its a concrete type ( def ) in a layer(Assembly_room_set) that has defined it.

Reply:
Ah, O.K., I guess I was misled by the test docstring, which still sounded relevant.

Reply:
I can't agree more. Indeed misleading.